server/tools/peertube-import-videos.ts

```diff
@@ -33,27 +34,64 @@ command
   .option('-U, --username <username>', 'Username')
   .option('-p, --password <token>', 'Password')
   .option('--target-url <targetUrl>', 'Video target URL')
+  .option('--tmpdir <tmpdir>', 'Working directory')
   .option('--since <since>', 'Publication date (inclusive) since which the videos can be imported (YYYY-MM-DD)', parseDate)
   .option('--until <until>', 'Publication date (inclusive) until which the videos can be imported (YYYY-MM-DD)', parseDate)
-  .option('-v, --verbose', 'Verbose mode')
+  .option('--first <first>', 'Process first n elements of returned playlist')
+  .option('--last <last>', 'Process last n elements of returned playlist')
+  .option('-v, --verbose [verbosity]', 'Verbosity, from 0 (only report errors) to 4 (for debugging), default is 2 (info)')
   .parse(process.argv)
 
 getServerCredentials(command)
   .then(({ url, username, password }) => {
     if (!program[ 'targetUrl' ]) {
-      console.error('--targetUrl field is required.')
+      log.error('--targetUrl field is required.')
 
       process.exit(-1)
     }
 
+    if (!program['tmpdir']) {
+      program[ 'tmpdir' ] = __dirname;
+    } else {
+      if (!existsSync(program[ 'tmpdir' ])) {
```

Comment:
If we use `existsSync`, we may not notice when the `tmpdir` have no write access.

What about using [`accessSync`](https://nodejs.org/dist/latest-v10.x/docs/api/fs.html#fs_fs_accesssync_path_mode)?

Reply:
...this... I'll change it.